tests/drivers/clock_control/fixed_clock/app.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Google, LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+/ {
+	test {
+		test_fixed_rate_clk0: clk {
+			compatible = "fixed-clock";
+			clock-frequency = <100>;
+			#clock-cells = <0>;
```

Comment:
clock-cells needs to be removed

Reply:
That's of course a mistake. Still needs #clock-cells = <0>, added back to binding. Good catch. Thanks!.